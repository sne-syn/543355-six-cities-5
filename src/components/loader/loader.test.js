import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './loader';

test(`Render Loader`, () => {
  const tree = renderer
    .create(<Loader />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
