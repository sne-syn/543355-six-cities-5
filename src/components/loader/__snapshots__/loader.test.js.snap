// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Loader 1`] = `
<div
  className="loader"
  id="loader"
>
  <div
    className="loadingio-spinner-eclipse-tdjcbvvh5xi"
  >
    <div
      className="ldio-hnl6tyhogo8"
    >
      <div />
    </div>
  </div>
</div>
`;
